src/plan/gencopy/gc_works.rs

```diff
@@ -186,15 +186,15 @@ impl<VM: VMBinding> GCWork<VM> for GenCopyProcessModBuf {
         if mmtk.plan.in_nursery() {
             let mut modified_nodes = vec![];
             ::std::mem::swap(&mut modified_nodes, &mut self.modified_nodes);
-            worker.scheduler().closure_stage.add(
-                ScanObjects::<GenCopyNurseryProcessEdges<VM>>::new(modified_nodes, false),
-            );
+            worker.scheduler().work_buckets[WorkBucketId::Closure].add(ScanObjects::<
```

Comment:
This code style [lines 189-193] looks a bit weird.

Reply:
Done